src/utils/hooks.tsx

```diff
@@ -307,6 +309,24 @@ export function useForumData(
       });
     }
   }
+  async function fetchVotes(): Promise<LoadingResult<ChainVoteEntry[]>> {
+    if (collectionId && forum.permission.readAndWrite && isSuccess(forumData)) {
+      try {
+        const fetchData = await forum.getVotes(collectionId);
+        if (fetchData) {
+          setForumData({...forumData, votes: fetchData});
```

Comment:
should remove this line, and call `setForumData()` only in the useEffect hook. Other than that LGTM